.gitignore

```diff
@@ -1,3 +1,6 @@
+.idea
+test.log
```

Comment:
Scripts in the repo do not produce test.log and lint.log files, so I don't see a reason for polluting a gitignore with it. I suggest just to ignore all log files (`*.log`) if it helps with dev process. Notice the alphabetical order.